### ISSUE TITLE
Stream to phone

### ISSUE THREAD

**Issue by amuttsch** [clementine-player/Android-Remote#20]
_From [alphadel...@gmail.com](https://code.google.com/u/107136459110240960141/) on October 05, 2013 23:04:51_

I am so happy using Clementine and being able to control it from my Android phone!  The new ability to download tracks and playlists to my phone is incredible!

The only thing that could make it better would be to be able to stream the current song in Clementine to my phone, playing it through my phone.  This way I could, for example, play music through my wifi network to a stereo system in a different room, far from my computer where Clementine is.  Since I can't fit that much music on my phone, this would be really handy.

There's a request on the Clementine issue tracker to stream to Icecast servers, but since this app can already download songs from Clementine, all it really needs to do is download the current track and play it on the phone, then delete it.  Maybe it could even cache it in RAM instead of writing it to /sdcard.

Thanks for this app and making Clementine even better!

_Original issue: http://code.google.com/p/clementine-remote-android/issues/detail?id=20_

**Comment by renatofdds** [comment]
Would love to have this feature too!

**Comment by jeje99** [comment]
Can't wait for that feature, if need beta tester. I'm In !

**Comment by fcastilloec** [comment]
I was about to open an issue asking for streaming support when I saw this.
I know that it's been said that this is in the TODO list of features, I just wanted to add something extra and not sure if here or opening a new issue would be best.
I think that once streaming is solved, it would be easier to also implement Chromecast support. This way I could stream my music to my TV or stereo system with HDMI. Chromecast support willl make this app above all, since I don't know of any app that focuses on music that can do this.

**Comment by renatofdds** [comment]
@fcastilloec i totally agree. Getting  streaming support along with chromecast support would make clementine a killer app.

**Comment by Snevzor** [comment]
+1

**Comment by fcastilloec** [comment]
Has there been any development on this enhancement? It's almost 2.5 years since this was originally posted. Is there a roadmap of bugs/features that we can expect to see from this app?

**Comment by jotomo** [comment]
As an alternative/workaround, there's the option of having Pulseaudio send the audio via Wifi and using https://github.com/kaytat/SimpleProtocolPlayer (FOSS) to play back the stream This streams whatever audio is running on the box. I use it to combine my wired headphones and mobile into wireless headphones via this 'audio bridge' to play back music via clementine, movies via mpv and whatnot. 
A similiar application with more options (but not FOSS) is SoundWire http://www.georgielabs.net/
HTH.

**Comment by fcastilloec** [comment]
@jotomo Thanks for letting me know about this app, sound great for the original post, but I'm also looking for something that I can use with my Chromecast. I guess it's better if I just open a new issue specific to Chromecast, even though it might be extremely dependent on this issue being solved.

**Comment by ikarosilva** [comment]
+1
